Enterprise-CMCS/macpro-mdct-mfp
# Create database seed script

### Description

This PR adds a script `yarn db:seed` which brings up interactive prompts to create a WP, SAR, or Banner. It's based on the Postman Collection with a few enhancements:
- Logs user in via env variables and handles submitting API token with each request
- Uses Faker to randomize data
- Allows directly creating a WP or SAR with a specific status (e.g. submitted, archived)

![Screenshot 2024-06-04 at 7 59 49 AM](https://github.com/Enterprise-CMCS/macpro-mdct-mfp/assets/169298580/9cb2fbe8-2fcc-44d7-bf70-d04baaed347c)
![Screenshot 2024-05-30 at 2 39 30 PM](https://github.com/Enterprise-CMCS/macpro-mdct-mfp/assets/169298580/f7160bbd-fa65-48f2-ba4b-b43830d55d5b)

### Related ticket(s)


CMDCT-3676

---
### How to test

1. Update env with `SEED_*` envs - use `./run update-env`
2. Run MFP locally
3. In a new terminal, run `yarn db:seed`
4. Select an option to create a WP, SAR, or Banner
5. Log in to MFP locally and verify what you selected was created

### Notes




---
### Pre-review checklist
